app/services/working_day_calculator.rb

```diff
@@ -17,10 +17,18 @@ def initialize(date)
     @date = date
   end
 
+  def working_days_from_date(number)
+    number.positive? ? add_working_days(number) : subtract_working_days(number)
+  end
+
   def add_working_days(number)
     calendar.add_business_days(date, number)
   end
 
+  def subtract_working_days(number)
```

Comment:
I think this new functionality needs it's own test. At the very least, I would suggest updating `spec/services/working_day_calculator_spec.rb` so that `number_to_target_dates` includes a negative number. So for example:

```ruby
let(:two_working_days_before) { Date.parse('13-Dec-2018') } # Weekend skip 
let(:number_to_target_dates) do
  {
    3 => three_working_days_later,
    5 => five_working_days_later,
    6 => six_working_days_later,
    -2 => two_working_days_before
  }
end
```